feed-service/src/main/java/com/nashtech/feedservice/helper/ExcelHelper.java

```diff
@@ -3,23 +3,22 @@
 import com.nashtech.feedservice.model.Nasher;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.apache.poi.ss.usermodel.Cell;
-import org.apache.poi.ss.usermodel.Row;
-import org.apache.poi.ss.usermodel.Sheet;
-import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.*;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 import org.springframework.web.multipart.MultipartFile;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.util.*;
 
 public class ExcelHelper {
     private static final Logger logger = LogManager.getLogger(ExcelHelper.class);
     private static final String TYPE = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
```

Comment:
please revert the Header as previously used.

Reply:
Done